user-guide/Advanced_Functionality/Databases/Indexing_database/Elasticsearch/storage_per_DMA/Profile_migration_to_elastic.md

```diff
@@ -0,0 +1,103 @@
+> - You can cancel the migration process by clicking the *Cancel Migration* button in the *ProfileManager XML to Elastic* section.
+
+### Troubleshooting
+
+If the migration should fail for any reason, the migration status object in the SLNetClientTest tool window will get a red background color. The ``SLMigrationManager.txt`` and ``SLProfileManager.txt`` log files will contain more information.
+
+If a ``MigrationStatus`` is stuck in the ``InProgress`` state, you will need to cancel the migration and trigger the migration again. You can do so with the *Cancel Migration* button in the *ProfileManger XML to Elastic* section of the SLNetClientTest tool window.
+
+### Behavior in new installations
+
+When a new DataMiner Agent is installed, the used storage type will depend on when Profile Manager starts up for the first time.
+
+- If DataMiner is installed with the 10.0 installer and Profile Manager is used, XML storage will be used. An indexing database is not yet supported as a storage type for profiles in DataMiner 10.0. After you have upgraded this DataMiner Agent to DataMiner 10.0.8 or later, it will continue to use XML storage until you trigger the migration.
+
+- If DataMiner is installed on version 10.4.0 and Profile Manager is used, indexing database storage will be used. This means profiles will only be functional on a DataMiner System that has indexing database storage running.
+
+- When you add a new DataMiner Agent to an existing cluster, the same behavior applies as mentioned above.
+
+## Checking the storage type used by a DataMiner Agent
+
+If you want to know which storage type a DataMiner Agent is currently using, open the migration window as detailed under [Migrating from XML to the indexing database](#migrating-from-xml-to-the-indexing-database).
+
+The current storage type is displayed in the *ProfileManger XML to Elastic* section, under the *Cancel Migration* button stating "Currently using ..." .
+
+## Limitations and differences with XML storage
+
+### Difference when deleting profiles
+
+When XML storage is used, it is not possible to remove a profile when one of the DataMiner Agents in the cluster cannot be reached, as this could cause syncing issues. No such restrictions apply when indexing database storage is used.
+
+### Field size
+
+Field names in the indexing database have a maximum length of 32766 bytes, which means any field of a profile indexed in the database can only contain 32766 bytes. This limit is mostly important for string fields, which can contain 32766 characters of one byte (or 16383 characters of two bytes).
+
+> [!NOTE]
+> If there is an attempt to save a profile with a field that is too big, the API will return an ``UnknownError``. The ``SLProfileManager.txt`` log file will contain the actual exception, which will mention the field that could not be indexed in the database.
```

Comment:
We should probably test if 'UnknownError' this is the case for Profiles as well. I think it will be reason 'Unknown' (from the 'ManagerStoreError.Reason').